Hearthstone Deck Tracker/Controls/DeckImportingControl.xaml.cs

```diff
@@ -98,6 +99,20 @@ private void UpdateContent()
 			OnPropertyChanged(nameof(StartButtonVisibility));
 		}
 
+        private void onImportAll_Modified(object sender, RoutedEventArgs e)
+        {
+            if (ItemsControl != null)
+            {
+                for (int i = 0; i < ItemsControl.Items.Count; i++)
+                {
+                    ContentPresenter c = (ContentPresenter)ItemsControl.ItemContainerGenerator.ContainerFromItem(ItemsControl.Items[i]);
+                    CheckBox chk = c.ContentTemplate.FindName("CheckBoxImport", c) as CheckBox;

```

Comment:
Using `as` instead of casting implies it can be null. You should either change this to a cast or nullcheck `chk`.